### ISSUE TITLE
UI - Listings Page - listing displays (vertical alignments)

### ISSUE THREAD

**Issue by LiteBit** [particl/particl-desktop#974]
# Visual Balance of listings

When listings appear in listing view some descriptions and/or pictures are longer (more vertical) than others, making for an awkward looking grid.

Grid should be responsive horizontally and vertically. 

![image](https://user-images.githubusercontent.com/6236019/39714766-16e13e3a-51f9-11e8-8000-bef1d7868a11.png)

# More information

- Operating system and version: Windows 10
- Client version: 1.1.0-RC1-alpha

**Comment by HenkSwardt** [comment]
Hi @LiteBit 

What happens if you hit retry?

Regards

**Comment by HenkSwardt** [comment]
Hey @AllienWorks 

Can we have a look into this please. 

Thanks you

**Comment by AllienWorks** [comment]
This is a known "issue". Current layout doesn't allow much anything else. We could implement some grid library as [Masonry](https://masonry.desandro.com/), but I think the better approach to this would be to have standardized thumbnails (square-ish) and ability to crop the photos when creating new listings.

Tl;dr we know about this, but will be focusing on it after alpha. It needs work that isn't really essential for MVP.

**Comment by zaSmilingIdiot** [comment]
The cropping of uploaded images is the preferred and correct solution for this. In the meanwhile, a UI fix for this has been applied in https://github.com/particl/particl-desktop/pull/1234
